custom_typings/yeoman-environment.d.ts

```diff
@@ -4,7 +4,7 @@ declare module 'yeoman-environment' {
     registerStub(generator: any, namespace: string);
     run(target: string, options: any, done: Function);
     getGeneratorsMeta(): YeomanEnvironment.GeneratorMeta[];
-    lookup(callback: () => void);
+    lookup(callback: (error?: Error) => void);
```

Comment:
Agreed, I'm usually too trusting when it comes to this.

[Their official documentation](http://yeoman.io/environment/Environment.html#lookup) reads "Take err as first parameter." Given the `err` name and the fact that "error-as-a-first-argument" is the default node callback style, I'm confident. Do you still have concerns?